tests/common/aws_test_runner.c

```diff
@@ -98,38 +98,6 @@ static void RunTestSuites( void )
         RUN_TEST_GROUP( GGD_Helper_System );
     #endif
 
-    #if ( testrunnerFULL_SHADOW_ENABLED == 1 )
-        RUN_TEST_GROUP( Full_Shadow_Unit );
-        RUN_TEST_GROUP( Full_Shadow );
-    #endif
-
-    #if ( testrunnerFULL_SHADOWv4_ENABLED == 1 )
-        RUN_TEST_GROUP( Shadow_Unit_Parser );
-        RUN_TEST_GROUP( Shadow_Unit_API );
-        RUN_TEST_GROUP( Shadow_System );
-    #endif /* if ( testrunnerFULL_SHADOWv4_ENABLED == 1 ) */
-
-    #if ( testrunnerFULL_MQTTv4_ENABLED == 1 )
-        RUN_TEST_GROUP( MQTT_Unit_Validate );
-        RUN_TEST_GROUP( MQTT_Unit_Subscription );
-        RUN_TEST_GROUP( MQTT_Unit_Receive );
-        RUN_TEST_GROUP( MQTT_Unit_API );
-        RUN_TEST_GROUP( MQTT_Unit_Metrics );
-        RUN_TEST_GROUP( MQTT_System );
-    #endif /* if ( testrunnerFULL_MQTTv4_ENABLED == 1 ) */
-
-    #if ( testrunnerFULL_MQTT_STRESS_TEST_ENABLED == 1 )
-        RUN_TEST_GROUP( Full_MQTT_Agent_Stress_Tests );
-    #endif
-
-    #if ( testrunnerFULL_MQTT_AGENT_ENABLED == 1 )
-        RUN_TEST_GROUP( Full_MQTT_Agent );
```

Comment:
why do we remove MQTT_Agent test group?

Reply:
This is the old, old MQTT library from even before the CSDK v4 one. It's unrelated to the coreMQTT Agent library